table/index_builder.cc

```diff
@@ -24,21 +25,22 @@ namespace rocksdb {
 IndexBuilder* IndexBuilder::CreateIndexBuilder(
     BlockBasedTableOptions::IndexType index_type,
     const InternalKeyComparator* comparator,
+    const SliceTransform* slice_transform,
```

Comment:
Actually I realized that after separating index and filter builders prefix_extractor is not need anymore, so i am going to remove it too.